### DISCUSSION TITLE
[Feature] All AI based search and Faces should work for other accounts when "Partner Sharing" is used

### DISCUSSION THREAD

**Discussion by IIIDPGIII** [immich-app/immich#6024] [Feature Request]
### The feature

All AI based search and Faces should work for other accounts when "Partner Sharing" is used. Currently, when I share the family photos library, other users (family members) lose all the great powerful search features available to the main uploader account.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by Kaaybi** [comment]
Just for information, there was an issue about this: https://github.com/immich-app/immich/issues/2478

**Reply by Kaaybi** [reply]
Also might be a duplicate to https://github.com/immich-app/immich/discussions/5089

**Reply by jhf2442** [reply]
@Kaaybi Agree that this is a duplicate of #5089 , but the issue #2478 was closed and is IMHO not solved. How to proceed ?

**Reply by Kaaybi** [reply]
@jhf2442, it looks like no one answered anymore in the issue and it got eventually closed due to being stale!
We can either ask to reopen the issue or create a new one, we can also wait for a dev to come across https://github.com/immich-app/immich/discussions/5089

**Reply by jhf2442** [reply]
OK, I've asked to reopen it - I assume the devs focus rather on issues than in feature requests ?

**Reply by martabal** [reply]
This is not an issue but a much more complex feature than it looks like. No need to open / re-open the issue; everyone want that feature to be implemented at some point
